.circleci/config.yml

```diff
@@ -64,6 +64,24 @@ jobs:
           destination: reports
       - store_test_results:
           path: app/build/test-results
+      - save_cache:
+          name: Save node_modules cache
+          key: 'node_modules/v1/{{ arch }}/{{ checksum "/.circleci/package.json" }}'
+          paths: [ "node_modules" ]
+      - restore_cache:
+          name: Restore node_modules cache
+          keys:
+            - 'node_modules/v1/{{ arch }}/{{ checksum ".circleci/package.json" }}'
+            - 'node_modules/v1/{{ arch }}'
+      - run:
+          name: chmod permissions
+          command: |
+            chmod +x scripts/metrics.sh
```

Comment:
you wouldn't need to run this at every build. you just need to run it once locally and commit `metrics.sh` to repo